src/screens/account-info.tsx

```diff
@@ -15,7 +16,7 @@ export const AccountInfoScreen = () => {
   const navigation = useTypedNavigation();
   const wallet = useWallet(route.params.accountId);
   const [balance, setBalance] = useState(
-    app.getBalance(route.params.accountId),
+    formatBalanceToNumber(app.getBalance(route.params.accountId)),
```

Comment:
IMHO there better format balance throw useMemo function

Reply:
Nope, this is initial value and it will be calculated here only once